automation/utils.py

```diff
@@ -2,10 +2,15 @@
 import logging
 import os
 import sys
+from typing import List, Optional, Iterable, TypeVar, Callable, Union
 
 from crontab import CronTab
 import yaml
 
+if sys.version_info[0] == 3 and sys.version_info[1] >= 8:
+    from typing import TypedDict, Literal
+else:
+    from typing_extensions import TypedDict, Literal
 
 DEV_ENVIRONEMT =  os.environ.get("DEV_ENVIRONMENT") == '1'
 APP_NAME = "SatNOG-PG"
```

Comment:
I'm not fan of the capitalized name. This typically should match the filename of the tool. If we want to use the same name everywhere, I'd go with lowercase: satnogs-gdn or satnogs-gut. Much easier to type the config file path.

Reply:
Fixed.